From 634fd658c3379e5f104cbd6317d6b849fc12be5e Mon Sep 17 00:00:00 2001
From: Yan Markman <ymarkman@marvell.com>
Date: Sun, 13 Jan 2019 08:24:07 +0200
Subject: [PATCH 0892/1051] arm:dts: a3720 model string with no DDR3
 specification

The Kernel's DTB is not responsible for DDR selecting and is the
same for boards with DDR3 and DDR4.
Remove the sub-string DDR3 from the Board-model description.

Change-Id: I17ff1f0029615f903185d083bec1135feecc1fca
Signed-off-by: Yan Markman <ymarkman@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/2029
Tested-by: sa_ip-sw-jenkins
Reviewed-by: Stefan Chulski <Stefan.Chulski@cavium.com>
[Kevin: The original patch got from Marvell sdk10.0_19.06]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-3720-db.dts | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-3720-db.dts b/arch/arm64/boot/dts/marvell/armada-3720-db.dts
index b8bc8cccb146..b1c132090fe5 100644
--- a/arch/arm64/boot/dts/marvell/armada-3720-db.dts
+++ b/arch/arm64/boot/dts/marvell/armada-3720-db.dts
@@ -1,7 +1,7 @@
 // SPDX-License-Identifier: (GPL-2.0+ OR MIT)
 /*
  * Device Tree file for Marvell Armada 3720 development board
- * (DB-88F3720-DDR3)
+ * (DB-88F3720 with either DDR3 or DDR4)
  * Copyright (C) 2016 Marvell
  *
  * Gregory CLEMENT <gregory.clement@free-electrons.com>
@@ -17,7 +17,7 @@
 #include "armada-372x.dtsi"
 
 / {
-	model = "Marvell Armada 3720 Development Board DB-88F3720-DDR3";
+	model = "Marvell Armada 3720 Development Board DB-88F3720";
 	compatible = "marvell,armada-3720-db", "marvell,armada3720", "marvell,armada3710";
 
 	chosen {
-- 
2.17.1

